tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -28,9 +28,9 @@
 #endif
 }
 
-// [[OutputRegex, Must copy into same size]]
-[[noreturn]] SPECTRE_TEST_CASE("Unit.DataStructures.DataVector.RefDiffSize",
-                               "[DataStructures][Unit]") {
+    // [[OutputRegex, Must copy into same size]]
```

Comment:
what happened here?

Reply:
`clang-format` often gets error tests wrong in my experience :(

Reply:
Should be able to get around this by using:
```cpp
// clang-format off
// [[OutputRegex, ...]]
SPECTRE_TEST_CASE() {
  // clang-format on
```